modules/serverless-sinkbinding-odc.adoc

```diff
@@ -0,0 +1,57 @@
+// Module included in the following assemblies:
+//
+// serverless/knative_eventing/serverless-sinkbinding.adoc
+
+[id="serverless-sinkbinding-odc_{context}"]
+= Create a SinkBinding using the Developer perspective
+
+This guide describes the steps required to create, manage, and delete a sample SinkBinding instance by using **Developer** perspective in the {product-title} web console.
+
+.Prerequisites
+
+* You have Knative Serving and Eventing installed.
+* You have switched to the *Developer* perspective in the web console.
+
+.Procedure
+
+. Create the `event-display` sample Service to use as an event sink.
+.. Switch to the `knative-serving` **Project** in the dropdown menu.
+.. Navigate to **+Add** -> **Container Image**.
++
+image::serverless-odc-sinkbinding-1.png[Add a sample Service event sink as a Container Image]
++
+.. For the **Image** configuration, select **Image name from external registry**, then enter the sample URL:
++
+----
+quay.io/openshift-knative/knative-eventing-sources-event-display:v0.13.2
+----
+// TODO: Needs an update to latest sample image for 1.8.0?
```

Comment:
I think we should update those references to be in line with current supported Knative version in Serverless. So, for 1.8.0 that would be: `quay.io/openshift-knative/knative-eventing-sources-event-display:v0.14.1`

We should do that not because, without changing that, it will not work, but old images have more vulnerabilities inside them. Or maybe we should use here `latest` version ([v0.16.0 currently](https://quay.io/repository/openshift-knative/knative-eventing-sources-event-display?tag=v0.16.0&tab=tags)) as it has only 2 High reported CVE instead of 13 High reported for [v0.14.1](https://quay.io/repository/openshift-knative/knative-eventing-sources-event-display?tag=v0.14.1&tab=tags).

Also, using v0.13.2 when there are newer ones look weird from outside perspective.

Reply:
Thanks, that's exactly what I was looking for.

Reply:
Maybe it would be better to open an issue about this image version. Should we update it in all various places, as it is scattered all over eventing docs, or use latest?

Reply:
The event display source is pretty stable, as confirmed by @matzew. I'm convinced that using `latest` shouldn't backfire at us. We just rely on output that this image is producing in logs.

Even if that output will change in some future version, I doubt that's a real problem. In that case we could simply adjust the output in docs to be in line. It's also quite possible that such thing will never happen.

I'd say it's your decision @abrennan89. Either use the same version as release to avoid confusion or just lust latest for event display image. I'm okay with both. I am slightly leaning towards using `latest` as simpler solution.